hasura.planx.uk/metadata/tables.yaml

```diff
@@ -1919,7 +1942,14 @@
           - team_slug
           - created_at
           - submitted_at
-        filter: {}
+        filter:
+          team:
+            members:
+              _and:
+                - user_id:
+                    _eq: x-hasura-user-id
+                - role:
+                    _eq: teamEditor
```

Comment:
Users with the `teamEditor` role can only access records which are associated with teams where they hold the `teamEditor` role.